src/redux/reducers/logOut.reducer.js

```diff
@@ -0,0 +1,19 @@
+import { browserHistory } from 'react-router';
+import * as types from '../actions/types/logOut.type';
+
+const initialState = {
+
+};
+
+export default function sessionReducer(state = initialState.session, action) {
+  switch (action.type) {
+    case types.LOG_IN_SUCCESS:
```

Comment:
@EmyRukundo, I am wondering what is the use of this action type: `LOG_IN_SUCCESS` since we have our login working as expected.

Reply:
Yes @eliemugenzi , as you said this login is working as expected, this is the action type when been able to logout successful and redirect to the homepage, and as you may have seen we also have logout, which takes place when user tried to logout but didn't work might because of the user was logged in